mempool/tx_pool.go

```diff
@@ -108,9 +108,9 @@ func (t *TxMempool) Put(id types.TransactionID, tx *types.Transaction) {
 }
 
 // Invalidate removes transaction from pool.
-func (t *TxMempool) Invalidate(id types.TransactionID) {
+func (t *TxMempool) Invalidate(txID types.TransactionID, layerID types.LayerID) {
```

Comment:
what layer even means in the context of the txpool.Invalidate ?

Reply:
actually, it doesn't make sense to add it here, I reverted it